.pre-commit-config.yaml

```diff
@@ -1,74 +1,75 @@
 exclude: 'src/pip/_vendor/'
 
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v6.0.0
-  hooks:
-  - id: check-builtin-literals
-  - id: check-added-large-files
-  - id: check-case-conflict
-  - id: check-toml
-  - id: check-yaml
-  - id: debug-statements
-  - id: end-of-file-fixer
-    exclude: WHEEL
-  - id: forbid-new-submodules
-  - id: trailing-whitespace
-    exclude: .patch
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
+    hooks:
+      - id: check-builtin-literals
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-toml
+      - id: debug-statements
+      - id: end-of-file-fixer
+        exclude: WHEEL
+      - id: forbid-new-submodules
+      - id: trailing-whitespace
+        exclude: .patch
 
-- repo: https://github.com/psf/black-pre-commit-mirror
-  rev: 25.9.0
-  hooks:
-  - id: black
+  - repo: https://github.com/google/yamlfmt
+    rev: v0.20.0
+    hooks:
+      - id: yamlfmt
+        types: [yaml]
 
-- repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.14.3
-  hooks:
-    - id: ruff-check
-      args: [--fix]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
+    hooks:
+      - id: check-yaml
 
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v1.18.2
-  hooks:
-  - id: mypy
-    exclude: tests/data
-    args: ["--pretty", "--show-error-codes"]
-    additional_dependencies: [
-        'keyring==24.2.0',
-        'nox==2024.03.02',
-        'pytest',
-        'types-docutils==0.20.0.3',
-        'types-setuptools==68.2.0.0',
-        'types-freezegun==1.1.10',
-        'types-pyyaml==6.0.12.12',
-        'typing-extensions',
-    ]
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 25.9.0
+    hooks:
+      - id: black
```

Comment:
Has this changed the order of the blocks? If we need to keep a specific order, that seems a bit more annoying than just fixing formatting.